ha_cluster_exporter.go

```diff
@@ -189,15 +183,23 @@ func main() {
 			sbdConfiguration, err := readSdbFile()
 			if err != nil {
 				log.Println(err)
+				time.Sleep(time.Duration(int64(*timeoutSeconds)) * time.Second)
 				continue
 			}
 			// retrieve a list of sbd devices
-			sbdDevices := getSbdDevices(sbdConfiguration)
+			sbdDevices, err2 := getSbdDevices(sbdConfiguration)
+			// mostly, the sbd_device were not set in conf file for returning an error
+			if err2 != nil {
```

Comment:
isn't common idiom in Go to just mask always the same `err` variable?

Reply:
yop

Reply:
I think i did atypo somewhere :grin: